src/r1cs/inner.rs

```diff
@@ -252,13 +283,9 @@ impl AllocVar<Element, Fq> for ElementVar {
                     mode,
                 )?,
             }),
-            AllocationMode::Input => Ok(Self {
-                inner: EdwardsVar::new_variable_omit_prime_order_check(
-                    cs,
-                    || Ok(group_projective_point.inner),
-                    mode,
-                )?,
-            }),
+            AllocationMode::Input => {
+                unimplemented!()
+            }
```

Comment:
non-blocking: it might be more ergonomic if this was filled in with (a) a call to `compress_to_field` on the provided `Element` followed by (b) forwarding to the `AllocVar<Fq, _>` implementation